internal/iam/repository_test.go

```diff
@@ -160,6 +160,7 @@ func Test_Repository_delete(t *testing.T) {
 		err = db.TestVerifyOplog(t, rw, s.PublicId, db.WithOperation(oplog.OpType_OP_TYPE_DELETE), db.WithCreateNotBefore(5*time.Second))
 		require.NoError(err)
 	})
+	oplog.TestOplogDeleteAllEntries(t, db.New(conn).UnderlyingDB())
```

Comment:
This is now needed in some integration tests whenever we want to delete all KMS keys, to avoid foreign key reference errors (the oplog has `restrict` on delete).